x-pack/solutions/search/plugins/search_connectors/kibana.jsonc

```diff
@@ -14,8 +14,7 @@
     "browser": true,
     "configPath": [
       "xpack",
-      "search",
-      "connectors"
+      "searchConnectors"
```

Comment:
This and the file below could be considered a breaking change. However, the only config available is to be able to disable the plugin. Since Kibana crashes with the previous config, I'm going to assume that they are not used.